### DISCUSSION TITLE
blockchain

### DISCUSSION THREAD

**Discussion by Marklord-dev** [Cyfrin/foundry-full-course-cu#1949] [Q&A]
Hi, umm, i'm wondering, what is the limit of transactions that can be stored on a block?

**Comment by EngrPips** [comment]
Hello @Marklord-dev, Honestly this might differ blockchain to blockchain but you should read this post `https://medium.com/ethereum-grid/ethereum-101-how-are-transactions-included-in-a-block-9ae5f491853f` to gain more insight about a blockchain block and how transactions are included in it.

**Reply by Marklord-dev** [reply]
Hi, EngrPips, thank you for the information, it was really helpful.

**Comment by urosognjenovic** [comment]
For Ethereum there's no explicit limit on the number of transactions that can be included in a block. However, there's a limit on the amount of gas per block (30 million). It means that all the transactions in a block can consume no more than 30 million gas.

**Reply by EngrPips** [reply]
Interesting
